blocks/editable/index.js

```diff
@@ -159,8 +159,8 @@ export default class Editable extends Component {
 	}
 
 	registerCustomFormatters() {
-		forEach( this.props.formatters, ( formatter ) => {
-			this.editor.formatter.register( formatter.format, formatter.formatter );
+		forEach( this.props.initialFormatters, ( formatter ) => {
+			this.editor.formatter.register( formatter.format, { ...formatter.formatter } );
```

Comment:
Why do we need a new instance? Also the `formatter.formatter` bothers me a bit, maybe we could deconstruct instead?

Reply:
We do in this instance as Tiny mutates the formatter and then we can't compare `this.props.initialFormatters` with `nextProps.initialFormatters` in `componentWillReceiveProps`.